Reworks the Steward's Office and Gatekeepers office.

powerfulXtian opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- Note: PRs including balance changes authored by anyone other than maintainers and official devs will not be considered. -->

## About The Pull Request

This PR reworks the Steward and Gatekeeper's places to swap them both in order to allow access to the outside without the the problem of constant traffic through the gatekeep.

Adds a few more keys to the Stewards office, including:
>Walls, Soilson, Tailor's, Forrest Garrison, Clinic.

Also adds a few more additions to the Steward's office, Including:
>Meeting Room
>Reworked Balcony
>Bookshelfs
>Additional Scrolls, Parchments, Feathers
>Reworked Bedroom
>Reworked Nervemaster Room
>Halfcloak, Raincloak, and Extra (empty) Pouch

Gatekeep has been reworked as well, new additions include:
>One additional Weaponspawner
>One additional Armorspawner
>Deck of cards and Die
>Two crossbows
>Two Bolt Quivers

Unfortunately, this rework minimizes the small park in the center, reducing it to two trees. The bench and waterfountain have also been deleted.
![image](https://github.com/user-attachments/assets/6ad8af70-6d8c-4271-8197-2cb6a3969f7e)
![image](https://github.com/user-attachments/assets/0bb29bff-5f8b-43f4-999c-6ef541ba2e95)

## Why It's Good For The Game

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

This rework opens up the Steward to be able to be normally accessible by the public, like the Merchant/Smithy. The Steward is supposed to coordinate the Keep's mammon, including handing out loans and key replacements. This task proved to be difficult with the constant traffic and hustle of the gate, hence the rework. The additional rooms the Steward has also opens up the door for more spaces to RP in. Hopefully with this update, the Steward will see more RP and fun.

The Gatekeep has also been reworked as a result of this, turning the atmosphere into a more shitty cramped area, as is expected of a gate. The common trend of weapon revoking is still honored here, with racks placed upstairs. Thieves now can break into the Gatekeep in a second way, through where previously how one would break into a Steward's Office. The racks stored up here allows thieves to also steal weaponry if the Gatekeep isn't attentive enough.

This update also reduces the amount of wells in Vanderlin to one. This makes acquiring water more tedious with the reduction of wells to the western side of the Town. Hopefully more usage of water barrels will come as a result of this.

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->

aberrantQuesrist commented:
I'm not a huge fan of the steward's office hanging out so far across the river, or the bridge leading up to it. Otherwise I think changing the sides is a good idea- could you retain the street light and bar area to the South of the (new) stewardry and remove the bridge, as well as shrink box that's jutting out?

powerfulXtian commented:
By shrinkbox, do you mean the balcony? Or where the nervemaster is located. A bit new to the lingo.

I can readd the Bars and Bridge, but I'm not quite sure how the Stewardry is going to function since... The bridge is what leads them to the entrance.

More details, please!

aberrantQuesrist commented:
My concern is largely aesthetic, yeah. I'm referring to the portion that hangs over the river. Balcony also isn't in a place I'm thrilled with, I'd prefer THAT hang over the river on the right hand side if you want to have it.

Why not have the countertop facing the pit? While yes, it can get congested, it also gives the stationed guards more interesting things to watch and/or intervene with.

powerfulXtian commented:
![image](https://github.com/user-attachments/assets/24d51a19-4a98-48fa-8fc9-5fec479e3518)
![image](https://github.com/user-attachments/assets/dbd2da96-cef5-4396-a4c2-163e64fd4a30)

changes

YosemiteYam commented:
The point of the stewardry being across the gatekeeper was so that you couldnt just break into the stewards to bypass the gate in a siege no? With this update the gate kind of loses its purpose against a large amount of people like rebels as they can just destroy the bars of the stewards counter and break in that way.

powerfulXtian commented:
shutters aren't breakable iirc

Box2444 commented:
can't you build a wall, climb over it and destroy the bars on the second floor here ?

powerfulXtian commented:
True. But at that point just let'm have it.

Box2444 commented:
IMHO this is the kind of stuff that needs to be put to the test for any oversights, but balancing is a concern at least in how much it changes the vulnerability of the keep. Pretty sure that without having a couple of rebellion rounds to put it to test it'll be hard to tell how different it is from the current map.

Eagletanker commented:
This basically fucks the keep. There's already not really a defensible wall, this removes the little bit of fortification it has. Plus, as a Steward Main, I don't really have people visiting my office enough to justify such large space, and as such I prefer the more cozier accommodations that exist right now. If I need to have a meeting, I'll just go to one of the meeting rooms within the keep.

Eagletanker commented:
If you want to give the steward better accomedations, add a table on the second floor, move the bed to where....you know what

Eagletanker commented:
give me a moment, what's your discord?

Eagletanker commented:
> My concern is largely aesthetic, yeah. I'm referring to the portion that hangs over the river. Balcony also isn't in a place I'm thrilled with, I'd prefer THAT hang over the river on the right hand side if you want to have it.
> 
> Why not have the countertop facing the pit? While yes, it can get congested, it also gives the stationed guards more interesting things to watch and/or intervene with.

Also, this, so much. I love being able to sit and talk with the gate guards when things are slow for the both of us. It makes the rounds go by faster and usually there's some good RP from it.

Eagletanker commented:
> This basically fucks the keep. There's already not really a defensible wall, this removes the little bit of fortification it has. Plus, as a Steward Main, I don't really have people visiting my office enough to justify such large space, and as such I prefer the more cozier accommodations that exist right now. If I need to have a meeting, I'll just go to one of the meeting rooms within the keep.

It doesn't feel like a fortified area that's serves a dual purpose in peace as the stewards office, but rather as a Stewards office that happens to be attached to a military fortification.

powerfulXtian commented:
>This basically fucks the keep. There's already not really a defensible wall, this removes the little bit of fortification it has.
The keep wall is pretty darn powerful already.  It has fortification if guards are manning bows. It falls apart quickly without guards. It should stay that way.

>This basically fucks the keep. There's already not really a defensible wall, this removes the little bit of fortification it has. Plus, as a Steward Main, I don't really have people visiting my office enough to justify such large space, and as such I prefer the more cozier accommodations that exist right now. If I need to have a meeting, I'll just go to one of the meeting rooms within the keep.
I'm trying to encourage more RP with this update and allowing them have more space to do so, and ways to do so mechanically. Having their office being able to be accessed from outside the gate was one of if not the biggest turnoffs for anyone wanting to interact with the Steward. Boom. Fixed. Want to take a meeting inside in your own PRIVATE place? Boom. Fixed.

>It doesn't feel like a fortified area that's serves a dual purpose in peace as the stewards office, but rather as a Stewards office that happens to be attached to a military fortification.
...Like the current Steward office that is a joint fusion with a gate?

>Also, this, so much. I love being able to sit and talk with the gate guards when things are slow for the both of us. It makes the rounds go by faster and usually there's some good RP from it.
You can still talk to them via the window at the keyroom. I made sure to not turn it into walls.

Honestly be open to change. You may see this as useless but the other 100 or so people in this game may find use for the new content as, being honest? Current office just ain't it.

powerfulXtian commented:
i quit mapping lol
